### DISCUSSION TITLE
AVX support

### DISCUSSION THREAD

**Discussion by Torinde** [ptitSeb/box86#972] [General]
[Intel AVX instructions will be available on both 32bit and 64bit flavors of Processor and OS](https://community.intel.com/t5/Intel-ISA-Extensions/Are-AVX-Intrinsics-supported-on-32-bit-processors/m-p/771334)
[VEX can be used in 32-bit mode with the some restrictions](https://en.wikipedia.org/wiki/VEX_prefix)

@ptitSeb, it seems you're adding AVX to box64, do you plan to keep both in sync, e.g. everything except x86-64?

**Comment by ptitSeb** [comment]
> [Intel AVX instructions will be available on both 32bit and 64bit flavors of Processor and OS](https://community.intel.com/t5/Intel-ISA-Extensions/Are-AVX-Intrinsics-supported-on-32-bit-processors/m-p/771334) [VEX can be used in 32-bit mode with the some restrictions](https://en.wikipedia.org/wiki/VEX_prefix)
> 
> @ptitSeb, it seems you're adding AVX to box64, do you plan to keep both in sync, e.g. everything except x86-64?

No, I don't plan to add AVX support to box86, because I see no actual use case for AVX in box86 (also, sidenote, box64 with wine+wow64 can allow you to use AVX in 32bits windows program, but I don't expect to be much of them).
